packages/style/components/queryBuilder/QueryPill.module.scss

```diff
@@ -20,7 +21,7 @@
     }
 
     .close {
-        padding: .3rem .2rem .3rem .4rem;
+        padding: 0 .2rem 0 .4rem;
```

Comment:
I styled the x close button with Lucas and this was what he wanted with th .3rem .2rem .3rem .4rem. why are we removing the .3rem ?

Reply:
if you look at the design, it breaks his own line-height